src/EnergyPlus/PlantUtilities.cc

```diff
@@ -149,7 +149,14 @@ void SetComponentFlowRate(EnergyPlusData &state,
     Real64 const MdotOldRequest = state.dataLoopNodes->Node(InletNode).MassFlowRateRequest;
     auto &loop_side = state.dataPlnt->PlantLoop(plantLoc.loopNum).LoopSide(plantLoc.loopSideNum);
     auto const &comp = loop_side.Branch(plantLoc.branchNum).Comp(plantLoc.compNum);
-
+    if (DataPlant::CompData::getPlantComponent(state, plantLoc).FlowCtrl == DataBranchAirLoopPlant::ControlType::SeriesActive &&
+        plantLoc.loopSideNum == DataPlant::LoopSideLocation::Supply) {
```

Comment:
added supply side to logic check so new behavior doesn't impact coils in series.  Found that in UnitarySystem, early calls to check the max capacity of a coil with full water flow were setting up flow that then would not get shut down with the change.